apps/meteor/tests/end-to-end/api/05-chat.js

```diff
@@ -1347,8 +1347,14 @@ describe('[Chat]', function () {
 	});
 
 	describe('[/chat.getMessageReadReceipts]', () => {
+		const isEnterprise = typeof process.env.IS_EE === 'string' ? process.env.IS_EE === 'true' : !!process.env.IS_EE;
 		describe('when execute successfully', () => {
 			it("should return the statusCode 200 and 'receipts' property and should be equal an array", (done) => {
+				if (!isEnterprise) {
+					done();
```

Comment:
Would be better to skip it, so we know it's a test that didn't run rather than run successfully

Reply:
Agree, done!